app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +19,8 @@
         android:theme="@style/Theme.OpenCVTest">
         <activity
             android:name=".MainActivity"
+            android:configChanges="orientation|screenSize|keyboardHidden"
+            android:exported="true">
```

Comment:
is this a goofy merge thing? duplicating the android:exported="true">

Reply:
I have no clue